internal/transform/claimable_balance.go

```diff
@@ -47,11 +47,13 @@ func TransformClaimableBalance(ledgerChange ingest.Change) (ClaimableBalanceOutp
 
 	transformed := ClaimableBalanceOutput{
 		BalanceID:          balanceID,
+		Asset:              balanceEntry.Asset,
```

Comment:
Do these new columns get added to tables in Bigquery? What is the reasoning behind adding the elements here?

Reply:
These elements were added to overcome types incompatibility between functions from `verify` package and `stellar-etl` internal types. Its `json:` struct tags were given the special `"-"` value that makes the `encoding/json` package ignore parsing those fields. All new fields with the `Raw` prefix follows the same logic so it answers the question above.